doc/options-reference.md

```diff
@@ -1159,6 +1160,25 @@ module.exports = {
 
 </details>
 
+## mermaid
+
+By default the `mermaid` reporter delivers "compressed" results - This means that the rendered appearance remains the same, but the node is hashed and shortened. The default value for mermaid.js limits the amount of text that mermaid.js will render to 50000 characters.
+However, it is also possible to output highly readable results without compression:
+
+```javascript
+module.exports = {
+  // ...
+  options: {
+    reporterOptions: {
+      mermaid: {
+        // Whether or not to compresses the output text. Defaults to true.
+        minify: false,
+      },
+    },
+  },
+};
+```
+
```

Comment:
I am not fluent in English, so I would be happy to point out any discrepancies!(You can add a commit and have it modified.)